src/debugAdapter/runtimeInterface.ts

```diff
@@ -3,21 +3,19 @@
 
 import truffleDebugger from '@truffle/debugger';
 import {EventEmitter} from 'events';
-import {filterContractsWithAddress, prepareContracts} from './contracts/contractsPrepareHelpers';
+import {prepareContract} from './contracts/contractHelpers';
 import {TranslatedResult, translateTruffleVariables} from './helpers';
 import {DebuggerTypes} from './models/debuggerTypes';
 import {ICallInfo} from './models/ICallInfo';
-import {IContractModel} from './models/IContractModel';
 import {IInstruction} from './models/IInstruction';
 
 export default class RuntimeInterface extends EventEmitter {
   private _isDebuggerAttached: boolean;
   private _session: truffleDebugger.Session | undefined;
   private _selectors: truffleDebugger.Selectors;
   private _numBreakpoints: number;
-  private _deployedContracts: IContractModel[];
   private _initialBreakPoints: Array<{path: string; lines: number[]}>;
-  private _fileLookupMap: Map<string, string>;
+  private _sources: Map<string, string>;
```

Comment:
From the signature, type+name, is not clear what this `Map` is about. Shall we either rename it, or add a doc comment?